concourse/resources/image.mako

```diff
@@ -10,3 +10,19 @@ credentials = registry_cfg.credentials()
     password: '${credentials.passwd()}'
     repository: ${image_reference}
 </%def>
+<%def name="task_image_resource(registry_cfg, image_repository, image_tag, indent=0)"
+filter="indent_func(indent),trim">
+<%
+from makoutil import indent_func
+# registry_cfg must be of type ContainerRegistryConfig (cc-utils)
```

Comment:
good idea to comment this. why not even add it as a type hint. or use util.check_type? or both